src/main/java/com/ceritoxi/util/Math.java

```diff
@@ -1,130 +1,457 @@
 
+public class Meth {
+
+    public static boolean isDividableByXorY(int dividend, int x, int y) {
+        if ((x != 0 && dividend % x == 0) || (y != 0 && dividend % y == 0)) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfNumbersUnderZThatAreDividableByXorY (int x, int y, int z) {
+        int sum = 0;
+
+        if (z < 1) {
+            return sum;
+        }
+        for (int i = 1; i < z; i++) {
+            if (isDividableByXorY(i, x, y)) {
+                sum += i;
+            }
+        }
+        return sum;
+    }
+
+    public static boolean isEven (int number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isEven (long number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfEvenFibonacciTermsUnderX (int x) {
+
+        int result = 0;
+        int sum = 0;
+        int firstTerm = 0;
+        int secondTerm = 1;
+
+        while (sum < x) {
+            if (isEven(sum)) {
+                result += sum;
+            }
+            sum = firstTerm + secondTerm;
+            firstTerm = secondTerm;
+            secondTerm = sum;
+        }
+        return result;
+    }
+
+    public static boolean isPrime (long number) {
+
+        if (number == 2 || number == 3 || number == 5) {
+            return true;
+        }
+        if (number <= 1 || number % 2 == 0 || number % 3 == 0 || number % 5 == 0) {
+            return false;
+        }
+
+        for (long i = 3; i <= java.lang.Math.sqrt(number); i += 2) {
+            if (number % i == 0) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    public static boolean isPrime (int number) {
+
+        if (number == 2 || number == 3 || number == 5) {
+            return true;
+        }
+        if (number <= 1 || number % 2 == 0 || number % 3 == 0 || number % 5 == 0) {
+            return false;
+        }
+
+        for (int i = 3; i <= java.lang.Math.sqrt(number); i += 2) {
+            if (number % i == 0) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    public static int nextPrime (int number) {
+        if (number == 0) {
+            return 2;
+        }
+        if (isEven(number) || number == 1) {
+            number++;
+        } else {
+            number += 2;
+        }
+        while (!isPrime(number)) {
+            number += 2;
+        }
+        return number;
+    }
+
+    public static long nextPrime (long number) {
+        if (number == 0) {
+            return 2;
+        }
+        if (isEven(number) || number == 1) {
+            number++;
+        } else {
+            number += 2;
+        }
+        while (!isPrime(number)) {
+            number += 2;
+        }
+        return number;
+    }
+
+    //assuming prime is indeed a prime
```

Comment:
Comments are usually useful until the next commit and directly deceptive and harmful after the third :).

I suggest not to make them at all. Either tell what you want in the function name, check for edge cases in the code, or if you really really want to add a comment add javadoc instead.
Its a special type of comment at the start of the function like this:
```
/**
* 
*/
public static int primeIndex (int prime) {...
```
Its generally more maintained and looked for (as it appears in IDE-s as the explanation for the function together with the functions header

Reply:
mokay